### DISCUSSION TITLE
Indicating functions in the manual

### DISCUSSION THREAD

**Discussion by TomSnijders** [stocnet/rsiena#71] [General]
Robert discussed a few months ago whether functions should be indicated with a terminal pair of parentheses (). I answered that this might be done or might be left out. I now decided negatively about the () and changed in the manual all functions to be reported without the (). By the way, I saw that somebody had made some changes automatically which led to double sets of parentheses, which I now also changed back.

**Comment by jhollway** [comment]
To use "Discussions" for just that: @RWKrause, what are the arguments for and against suffixing functions with `()`? Does it help e.g. `{pkgdown}` to recognise and link functions to their documentation on the website? Is it supposed to visually distinguish functions from arguments or objects? Does it aid or impede readability?

**Comment by RWKrause** [comment]
Hey
The idea was that functions are visually distinct from objects this way. However, I think most functions in RSiena use verbs anyway (setEffect(), include..., sienaAlgorithmCreate).
However, some functions create objects of a class that is the name of the function, e.g., coCovar() creates an object of the class coCovar. I think it would be easier to distinguish objects and arguments within functions from functions if the functions always had ( ).

Anyway, not a hill I need to die on. If you rather have no ( ) then I wont argue about it.
Cheers
Robert
